app/assets/javascripts/style.js

```diff
@@ -33,15 +33,4 @@ function setAsideHeight() {
   if (titleHeight < moduleHeight) {
     $('.set .title-outer-container').outerHeight(moduleHeight);
   }
-
-  // resize header on guide page
-  var moduleHeight = $('.guide .set-link .module').outerHeight();
-  var titleHeight = $('.guide .title-outer-container').outerHeight();
-
-  if (moduleHeight < titleHeight) {
-    $('.guide .set-link .module').outerHeight(titleHeight);
-  }
-  if (titleHeight < moduleHeight) {
-    $('.guide .title-outer-container').outerHeight(moduleHeight);
-  }
```

Comment:
This javascript resized the blue background box for the title.  Since the title no longer has a blue background, the javascript is no longer needed.